manifests/m/Mozilla/Firefox/98.0.2/Mozilla.Firefox.installer.yaml

```diff
@@ -27,6 +25,15 @@ Installers:
   InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/98.0.2/win64/en-US/Firefox%20Setup%2098.0.2.msi
   InstallerSha256: AC31A566E82736073AEBF230B9557AF1AB43514C7A5C2149DC78FF98912AA9C9
   ProductCode: '{1294A4C5-9977-480F-9497-C0EA1E630130}'
```

Comment:
```suggestion
  ProductCode: 'Mozilla Firefox 98.0.2 (x64 en-US)'
```
I mean change it to like this which is the real ProductCode for en-US Firefox x64

Reply:
Oh, that makes more sense.